### PR TITLE
Added the grimoire 

### PR DESCRIPTION
**Pull request by nightbe**
I hope this works

### CODE REVIEW

**Comment by GuillaumeDIDIER** [comment]
This worked !

**Comment by GuillaumeDIDIER** [comment]
Ok, so I'm going to be annoying but:

Spells description can just go in full in the sheet file.
Alternately, they can just go into the global grimoire. I don't think having an extra grimoire file (with every single TeFo listed even when empty) is a good idea.

File: Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+ReVi lvl 30  
+**R:** Arc, **D:** Con, **T:** Ind  
+You can open a magical channel from yourself to some target, allowing you to cast any spell with a range greater than Personal on that target. The tunnel does not, of itself, grant any sense impressions of the target — you must cast appropriate spells through it if you wish to see. Spells cast through the tunnel cannot be of higher level than that of this spell. You must make standard Concentration rolls to maintain concentration on the Opening spell as you cast
+other spells. A magus who magically recognizes the tunnel (through The Invisible Eye Revealed or a similar spell) may cast spells through the tunnel
+back at you, and he need not concentrate on keeping the tunnel open. If this magus is the target of the spell, he is considered to be touching you. Otherwise, his effective range to you is his range to the target of the tunneling spell. If the target of the spell has Magic Resistance, you must penetrate that resistance to open the tunnel. Spells cast through the tunnel must also penetrate Magic Resistance normally.  
+Spells lvl 30 or less can be cast through.    
+*(Base effect, +4 Arc, +1 Conc)*
+
+
+**Name**
```
Comment (GuillaumeDIDIER) [inline Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md]: No need to keep the template here

File: Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+# Aquam
+
+## Creo Aquam
+
+**Flamable lube never hurt nobody**  
+CrAq 10  
+**R:** Voice, **D:** Dia, **T:** Ind  
+Caster flics their finger and covers an individual in highly flamable oil. If it gets ignited it does +12 damage the first round, +6 damage the second round, +0 damage the third round.    
+**Mastery:** Multi
```
Comment (GuillaumeDIDIER) [inline Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md]: Mastery are properties of how a magi knows a spell, not of the spell itself, so it doesn't belong in a file distinct from the character.

Also, I think mastery needs level in the ability, xp, and then the list of masteries options (Multi-cast, imperturbable casting etc)

File: Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+The target gains a +6 bonus to Recovery rolls to recover from injuries or diseases, as long as he has been under the influence of this spell for the whole of the recovery interval. The recovery interval is counted from the time that the spell is cast; any previous time is ignored.  
+*(Base 3, +1 Touch, +3 Moon)*
+
+**The Healing leaves litte time for the Hurting**  
+CrCo lvl 20  
+**R:** Touch, **D:** Mom, **T:** Ind. Ritual  
+This spell heals a single Light Wound suffered by the person touched. This spell does not heal damage from poison or disease.  
+(Base 15, +1 Touch)  
+
+**Im not done with you yet
```
Comment (GuillaumeDIDIER) [inline Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md]: Missing `**`

File: Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+Increase one of a person’s Strength by one point, to no more than 0.  
+*(Base 30, Touch +1)*
+
+## Intellego Corpus
+
+## Muto Corpus
+
+## Perdo Corpus
+
+**Mr Hiccup**
```
Comment (GuillaumeDIDIER) [inline Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md]: Great name

File: Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+
+## Perdo Ignem
+
+**Cloak of Shadow**  
+PeIg lvl 3  
+**R:** Personal, **D:** Dia, **T:** Ind  
+The caster is enveloped in shadows making it somewhat more difficult to clearly distinguish details of the caster by sight.  
+*(Base 3)*  
+
+**Jackfrost says fuck you particularly**
```
Comment (GuillaumeDIDIER) [inline Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md]: Couldn't we find a substitute to profanity ?

File: Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+## Rego Vim
+
+**Counterspell**  
+ReVi lvl 10  
+**R:** Voice, **D:** Con, **T:** Ind  
+Suppress a spell cast by another with level less than or equal to half the (level + 5 magnitudes) of the Vim spell = lvl 20.      
+**Mastery:** Fast cast  
+*(Base 3, Voice +2, con +1)*
+
+**Aegis of the Hearth (20)**
```
Comment (GuillaumeDIDIER) [inline Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md]: Those a Core book spells, we shouldn't duplicate the info, except within the character sheet itself if it includes the description of every spell.

**Comment by GuillaumeDIDIER** [comment]
So the Aine - Grimoire.md should be removed. Original spells go in the global grimoire, and I saw you updated Aine.md with the spells.

**Comment by GuillaumeDIDIER** [comment]
How mastery should be formated imho
```
Mastery 4 : Acute Sense x 2, Multicast, Penetration
```

add `(xx xp)` after the level when you also keep the xp for abilities around, but that's not the case here

**Comment by nightbe** [comment]
I will change it "one last time" and properly add the mastery in the way you showed :)

**Comment by nightbe** [comment]
Thanks for the patience :)

**Pull request merged**